_studio/mfx_lib/encode_hw/hevc/linux/g9/hevcehw_g9_va_packer_lin.cpp

```diff
@@ -387,14 +387,16 @@ static bool FillCUQPDataVA(
 
     mfxU32 inputW = CeilDiv(HEVCParam.PicWidthInLumaSamples, inBlkSize);
     mfxU32 inputH = CeilDiv(HEVCParam.PicHeightInLumaSamples, inBlkSize);
+    inputW = mfx::align2_value(inputW, HEVCParam.LCUSize/inBlkSize);
+    inputH = mfx::align2_value(inputH, HEVCParam.LCUSize/inBlkSize);
 
     bool bInvalid = !mbqp || !mbqp->QP || (mbqp->NumQPAlloc < inputW * inputH);
     if (bInvalid)
```

Comment:
inputW * inputH is increased now, the check below can fail. mbqp buffer is not required to be aligned to LCU, it is only block aligned.